documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -58,214 +54,116 @@ from additional images.
-multiple auxiliary images that all reference the same volume, or if 
-you want to customize the command that is used to copy the
-files from an auxiliary image into its volume. You can
-use command customization to force "replace" behavior instead
-of merge behavior.
-
-#### Expected merge order
-
-By default, the files from multiple auxiliary images that share
-the same volume are merged. Specifically:
-
-- Files from later images in the merge overwrite same named files from earlier images.
-
-- The contents of overlapping directories from multiple images are combined.
+See [Configuration Example 3: Multiple Images](#configuration-example-3-multiple-images).
 
-The expected merge order for auxiliary images that share
-the same auxiliary image volume is:
+#### Model and WDT Install Homes
+If you are using auxiliary images, it normally should not be necessary to set `domain.spec.configuration.models.modelHome` and
+`domain.spec.configuration.models.wdtInstallHome`. The model and WDT install files you supply in the auxiliary image
+(see [source locations](#source-locations)) are always placed in the `/aux/models` and `/aux/weblogic-deploy` directories,
+respectively, in all WebLogic server pods. When auxiliary image(s) are configured, the operator automatically changes
+the default for `modelHome` and `wdtInstallHome` to match.
 
-- If you specify auxiliary images at different `serverPod` scopes,
-  and they all share the same volume, then the files from the
-  domain scope will be merged first, the cluster scope second,
-  and the server scope last.
-
-- If you specify multiple auxiliary images at the same scope, 
-  and they all share the same volume, then the files
-  will be merged in the order in which images appear
-  under `serverPod.auxiliaryImages`.
-
-__Note:__ If the results of a merge yield two or more
-differently named `domain.spec.configuration.model.ModelHome` model files in
-an auxilliary image volume, then refer to
-[Model file naming and loading order]({{< relref "/userguide/managing-domains/model-in-image/model-files#model-file-naming-and-loading-order">}})
-for the model files loading order.
-
-#### Performing replaces instead of merges
-
-If multiple auxiliary images share the same volume
-and you prefer that a particular same named directory from a later
-image completely replaces a directory from a previous
-image instead of combining the two directories,
-then you can customize the command that the second image uses to populate
-the shared volume in order to force a replace.
+{{% notice warning %}}
+If you set modelHome and wdtInstallHome to a non-default value,
+then the operator will ignore the WDT model and installation files from the auxiliary image(s).
+{{% /notice %}}
 
-For example, you can customize the 
-the second image's `serverPod.auxiliaryImage.command` field
-to first delete the directory that was already copied from the
-earlier image and then have it perform a normal copy:
+#### Configuration Examples
```

Comment:
You need text between two headings. You can say something like, "The following configuration examples illustrate each of the previously described sections." or "Review the following configuration examples."